packages/rabbitmq/src/TxSubmitWorker.ts

```diff
@@ -0,0 +1,182 @@
+import { Channel, Connection, Message, connect } from 'amqplib';
+import { TxSubmissionQueue } from './rabbitmqTxSubmitProvider';
+import { TxSubmitProvider } from '@cardano-sdk/core';
+import { ogmiosTxSubmitProvider, urlToConnectionConfig } from '@cardano-sdk/ogmios';
+
+/**
+ * Controller class for the transactions submission worker which gets
+ * transactions from RabbitMQ and forwards them to the ogmios server
+ */
+export class TxSubmitWorker {
```

Comment:
If you are speaking about [this RunnableModule class](https://github.com/input-output-hk/cardano-js-sdk/blob/ddcc9a7c9955c0030e99bf83e6cb14dc1b59de5d/packages/cardano-services/src/RunnableModule.ts#L8); that would create a circular dependencies issue, being it in the `@cardano-sdk/cardano-services` package.
I can move it in the `@cardano-sdk/core`...
Please tell me how to go on on this point.
Thank you

Reply:
Ah yes... 
We wouldn't want this in `core`, but it could be hoisted to a common package (later). Disregard this request, but you can use that class as a reference for the logger pattern. I'll capture this follow-up task

Reply:
I think it's too specific to be in core package. I'd say a new package, either:
- `util` that we already planned to create
- `services-util` focused on backend patterns/types/utils

Somewhat relatedly: we could also consider hoisting all backend packages to a new monorepo, having cardano-js-sdk packages as dependency. Might make it easier/more efficient from sre perspective, as it seems #245 will build entire monorepo instead of just `cardano-services` and it's deps. Thoughts @rhyslbw ?

Reply:
I'll make this a spike. I'd like to understand why individual packages trees cannot be built before considering separation like that. We gain a lot by having this in a single codebase, with backend services (accessible as a library or exe) being part of the kit.